.gitignore

```diff
@@ -87,3 +87,6 @@ ui/vault-ui-integration-server.pid
 # for building static assets
 node_modules
 package-lock.json
+
+# FoundationDB Go bindings
+vendor/foundation
```

Comment:
What is this here for? I assumed it was because go bindings were being put into vendor, but the script below doesn't mention this directory at all so I don't see any indication it's being added to the vendor directory (and I'd argue it's the wrong place for it since the vendor directory is automatically managed by govendor).

Reply:
Drats, I could have caught that last time around. This is a leftover from my early attempts, and it can go.